apollo-router/src/plugins/cache/tests.rs

```diff
@@ -152,6 +152,9 @@ impl Mocks for MockStore {
 
 #[tokio::test]
 async fn insert() {
+    let mut parser = apollo_compiler::parser::Parser::new();
+    let schema = parser.parse_schema(SCHEMA, "test.graphql").unwrap();
+    let valid_schema = Arc::new(schema.validate().unwrap());
```

Comment:
If you’re not changing `Parser` parameters you can use `Schema::parse_and_validate()`

Same a few more times below